pontoon/checks/admin.py

```diff
@@ -15,15 +15,19 @@ class FailedCheckModelAdmin(admin.ModelAdmin):
         "resource",
```

Comment:
We only show translation in the list view, I wonder why is that.

Note that we have the same problem on master.

Reply:
@PFischbeck Hey, just curious if you'd like to have a look into this.

No pressure, we can also fix it at a later occasion.

Reply:
@mathjazz I'm not sure what you mean. I can see the other attributes in the list view too:
![grafik](https://user-images.githubusercontent.com/2820802/117003295-f651a980-ace4-11eb-848d-381b6e036982.png)
And in the edit view, the translation is enough to infer all other attributes:
![grafik](https://user-images.githubusercontent.com/2820802/117003372-141f0e80-ace5-11eb-85b2-1b043579b9e3.png)
Or do you mean we should list other translation attributes, like the translation author?

Reply:
Interesting, that's how the list looks for me:

![Posnetek zaslona 2021-05-04 ob 14 48 20](https://user-images.githubusercontent.com/626716/117005664-d1ab0100-ace7-11eb-8706-071781b4cefd.png)

Reply:
That's weird. Are the other columns present in the HTML? Maybe this is only caused by translations containing special characters (newlines etc.).
Or maybe someone else can try reproducing?

Reply:
It turns out some translations are very long and they break the layout. So one just needs to scroll horizontally and the fields get revealed. :)

Sorry for the noise, this is outside the scope of this PR anyways.
https://bugzilla.mozilla.org/show_bug.cgi?id=1709788